docs/engineering/android-app/configuring/config-types/widget.mdx

```diff
@@ -160,6 +160,36 @@ primaryTextActions | The list of actions associated with the primary text. If th
 secondaryTextActions | The list of actions associated with the secondary text. If the list is empty, no actions will be displayed | | |
 overflow | Determines how the text should be truncated if it is too long to fit | | [TextOverflow.ELLIPSIS] |
 
+##How to use ACTIONABLE BUTTON
+Defines the configuration properties for a Actionable Button view, which is a view that displays button with text strings, it's status ,visibility mode, and if it is enabled. This view can also have actions associated with it.
+
+ ``` json
+ [
+   {
+     "viewType": "BUTTON",
+     "smallSized": true,
+     "text": "@{taskDescriptionWithStartDate}",
+     "status": "@{taskStatusColorCode}",
+     "visible": "true",
+     "enabled": "true",
+     "actions": []
+   }
+ ]
+```
+
+### Config properties of ACTIONABLE BUTTON
+|Property | Description | Required | Default |
```

Comment:
might be a good time where we add another column on what possible values that a property can use.
one such example is the ServiceStatus issue.
`DUE` | `OVERDUE` | `COMPLETED`

Reply:
I'm referring to the column headings

Reply:
default is more of "if the property is not defined, use x as the value"

Reply:
Oh, let me clarify, I mean column heading in this kind of way:

The column heading:
Property | Description | Required | Default | Values

The column below the "Values" heading would be:
DUE | OVERDUE | COMPLETED

Let me know if that makes sense

Reply:
yes